src/App.jsx

```diff
@@ -1,7 +1,17 @@
 import data from "./data.json";
+import {Album} from "./Components/Album/Album"
+import { Header } from "./Components/Header/Header"
+import "./index.css"
+
+
+
 
-console.log(data);
 
 export const App = () => {
```

Comment:
App.jsx isnt formatted as nicely as your other code (eg. indented code). But it is nice and small, you have structured this project so well, folders and separate css for each module, excellent job